src/Kernel32.Desktop/Kernel32+CharacterAttributesFlag.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="CharacterAttributesFlag"/> nested type.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// The character attributes for console input/output.
+        /// </summary>
+        [Flags]
+        public enum CharacterAttributesFlag
+        {
+            None = 0,
+
+            /// <summary>
+            /// Text color contains blue.
+            /// </summary>
+            FOREGROUND_BLUE = 0x0001,
+
+            /// <summary>
+            /// Text color contains green.
+            /// </summary>
+            FOREGROUND_GREEN = 0x0002,
+
+            /// <summary>
+            /// Text color contains red.
+            /// </summary>
+            FOREGROUND_INTENSITY = 0x0008,
```

Comment:
LOL. I spent a good couple of hours on this PR. But what drew my attention was probably that the type ended in Flag, but didn't have `[Flags]` enum. So I was trying to understand whether they were really flags or not, which I typically determine by studying the ordinal values of the enum members to see if they look like bit flags. This one did, except within three steps I noticed a missing one. So I did more research.